frontend/plugin-manager/PluginManager.cpp

```diff
@@ -0,0 +1,285 @@
+#include "PluginManager.hpp"
+#include "PluginManagerWindow.hpp"
+
+#include <OBSApp.hpp>
+#include <widgets/OBSBasic.hpp>
+#include <qt-wrappers.hpp>
+
+#include <QMessageBox>
+
+#include <nlohmann/json.hpp>
+#include <fstream>
+#include <algorithm>
```

Comment:
For a future cleanup perhaps, I believe we separate stdlib includes from other includes, and we alphabetize them. See other frontend C++ files for examples.